pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -124,9 +118,14 @@ func getBinding(iface *v1.Interface, domain *api.Domain) (BindMechanism, error)
 	}
 
 	if iface.Bridge != nil {
-		vif := &VIF{Name: podInterface}
+		vif := &VIF{Name: podInterfaceName}
 		populateMacAddress(vif, iface)
-		return &BridgePodInterface{iface: iface, vif: vif, domain: domain, podInterfaceNum: podInterfaceNum}, nil
+		return &BridgePodInterface{iface: iface,
+			vif:                 vif,
+			domain:              domain,
+			podInterfaceNum:     podInterfaceNum,
+			podInterfaceName:    podInterfaceName,
+			bridgeInterfaceName: fmt.Sprintf("br-%s", podInterfaceName)}, nil
```

Comment:
Could we adopt something like a common `k6t` prefix for all interfaces which we as KubeVirt create inside a pod?

Reply:
No problem I will change the prefix to k6t on the bridges.